siswrap/wrapper_services.py

```diff
@@ -174,8 +200,7 @@ def run(self):
                 proc = subprocess.Popen(["/bin/sleep", "1m"])
                 exec_string = "/bin/sleep 1m"
             else:
-                exec_string = ExecString(self, self.conf_svc,
-                                         self.info.runfolder).text
+                exec_string = self.get_exec_string()
```

Comment:
Well - first I called the `exec_string`, but since that was the name of the variable it didn't fly... ;) But I guess I could just make it a function with that name and call it directly when creating the process. However I think that it is slightly less readable.